src/styles/styles.css

```diff
@@ -11,7 +12,7 @@ body {
 .settings {
   width: 100%;
   min-height: 20%;
-  background: rgba(255, 255, 255, 0.6);
+  max-height: 100%;
```

Comment:
I think on mobile it should have height: 100%;

Reply:
I'm unable to set all height and overflow settings by myself so I have created a separate issue for this: #34